punx/tests/test_nxdl_manager.py

```diff
@@ -7,29 +7,78 @@
 from .. import FileNotFound
 from .. import InvalidNxdlFile
 from .. import nxdl_manager
+from .. import nxdl_schema
 
 
-def raises_exception(text):
-    try:
-        lxml.etree.fromstring(text)
-    except lxml.etree.XMLSyntaxError as exc:
-        raise exc
-    except Exception as exc:
-        raise exc
-    else:
-        raise No_Exception
+def get_NXDL_definition(nxclass, file_set):
+    """
+    Return instance of nxdl_manager.NXDL__definition from file_set.
+
+    PARAMETERS
+
+    nxclass str:
+        Name of NeXus class (such as NXdata, NXentry, NXsas, ...)
+    file_set str:
+        Name of NeXus definitions *file set* to be found in
+        one of the local caches.  A file set is a directory
+        containing all the NXDL files and XML Schema files
+        that define a specific version of the NeXus definitions.
+    """
+    cache_manager.CacheManager()
+    manager = nxdl_manager.NXDL_Manager(file_set)
+    return manager.classes.get(nxclass)
+
+
+def navigate_path(path_start, nxpath):
+    """
+    Drill down the nxpath to the group containing the last item.
```

Comment:
"Drill down"? Do you mean truncate? clip?

Reply:
Rewording